.github/workflows/changelog.yml

```
@@ -0,0 +1,27 @@
+name: Update documentation CHANGELOG
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - CHANGELOG.md
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Add file to filesystem
+      run: |
+        git clone https://github.com/pestphp/website pestphp-website
+        cp CHANGELOG.md pestphp-website/source/docs
+      shell: bash
```

Just wondering if it's worth using the `actions/checkout` action to clone the website repository for consistency. 🙂 And for copying the changelog, it doesn't require the `shell` key as this is the default on `ubuntu-latest`. 👍
```suggestion
    - uses: actions/checkout@v2
    - name: Checkout website repository
      uses: actions/checkout@v2
      with:
        token: ${{ secrets.CHANGELOG_KEY }}
        repository: pestphp/website
        path: website
    - name: Copy CHANGELOG to website repository
      run: cp CHANGELOG.md website/source/docs
```

---

Yeah have you tested it?

---

Yep, I've just done a test (also updated my suggestion comment):
- [Test repo with the action workflow](https://github.com/owenvoke/changelog-test-action)
- [Test repo to emulate the website repo](https://github.com/owenvoke/changelog-test)

Here's the action run: https://github.com/owenvoke/changelog-test-action/runs/735251194

And the PR: https://github.com/owenvoke/changelog-test/pull/1